src/library/baseplaylistfeature.cpp

```diff
@@ -478,6 +484,20 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportTrackFiles() {
+    int rows = m_pPlaylistTableModel->rowCount();
```

Comment:
This has the same "could have an active search" issue -- did you want to make a new table model here?